CHANGELOG.adoc

```diff
@@ -28,17 +26,15 @@ All of its remaining content was only relevant inside of Sirius Web and it has t
 
 === Dependency update
 
-
 === Bug fixes
 
-
 === New Features
 
-
 === Improvements
 
 - [releng] Add some automatic checks to simplify code reviews
-
+- https://github.com/eclipse-sirius/sirius-web/issues/2161[#2161] [form] Improves borderStyle customization possibilities by adding the following CSS properties:
```

Comment:
Those are not related to border style. Those are layouting options for the flex box container. Margin, padding, gap, alignItems and justifyContent are not related to border at all in CSS. And please stop changing the empty lines in other parts of the CHANGELOG, you are creating needless conflicts.

Reply:
Customization is for flexbox only and the properties are not link to border anymore